javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -621,6 +626,13 @@ predicate returnedPropWrite(Function f, DataFlow::SourceNode base, string prop,
   base.flowsToExpr(f.getAReturnedExpr())
 }
 
+/**
+ * Holds if `f` may return `base`, which has a write of property `prop` with right-hand side `rhs`.
+ */
+predicate receiverPropWrite(Function f, string prop, DataFlow::Node rhs) {
```

Comment:
I moved this as well as `returnedPropWrite` into `FlowSteps.qll` so they are private and can be shared with `TrackedNode.qll`.

Technically this is breaking since `DataFlow::returnedPropWrite` was public and not marked as internal. But it's kind of an obscure thing to try and use from the outside, so rather than deprecating it I think a change note should suffice in this case, WDYT?

Reply:
Definitely; that predicate being public was an oversight.